app/users/routes.js

```diff
@@ -5,8 +5,9 @@ module.exports = [
   { name: 'new', pattern: '/users/new', handler: handlers.new_user },
   { name: 'list', pattern: '/users', handler: handlers.list_users },
   { name: 'details', pattern: '/users/:id', handler: handlers.user_details },
-  { name: 'edit', pattern: '/users/:id/edit', handler: handlers.user_edit },
+  { name: 'edit', method: 'POST', pattern: '/users/:id/edit', handler: handlers.user_edit },
```

Comment:
I haven't seen this in AGES, so maybe I can't remember correctly, but wasn't the `GET users.edit` route for "I want to edit a user, show me the edit form"?

So maybe we need another route for the actual update where user submit the update data? e.g. `PUT users.update`?

Reply:
The forms for editing the user are on the `users.details` page. Those forms POST to the `users.edit` handler. There was no user edit page, only a placeholder which showed a JSON dump of the user object.

Reply:
I see. Well, maybe it was just a Rails convention but naming this route `update` instead of `edit` probably would avoid any confusion as user is being _updated_ by the request.

Also - sorry I'm not trying to annoy you with this comment - shouldn't the method be `PUT` as users are updating not creating a resource. Not the end of the world if doesn't change of course: Computers don't care :D

Reply:
The frontend isn't a REST API, but if it was I would agree that `PUT` would be the correct verb. `update` is a better name for the handler 👍